pkgs/applications/misc/solaar/default.nix

```diff
@@ -0,0 +1,47 @@
+{fetchurl, stdenv, makeWrapper, gtk3, pygobject3, pyudev}:
+let
+  version = "0.9.2";
+in
+stdenv.mkDerivation {
+  name = "solaar-${version}";
+  src = fetchurl {
+    sha256 = "0954grz2adggfzcj4df4mpr4d7qyl7w8rb4j2s0f9ymawl92i05j";
+    url = "https://github.com/pwr/Solaar/archive/${version}.tar.gz";
+  };
+
+  buildInputs =  [
+    gtk3
+    pygobject3
+    pyudev
```

Comment:
So, why we want to mix packages and nix internals and we don't want to mix nix internals and packages with python packages?